docs/API.md

```diff
@@ -91,13 +91,15 @@ The following datatypes are used throughout this document:
     * forgotPasswordVerified    : `POST /passwordForgotToken/:id/verified`
 * Unverified tokens:
     * verifyTokens              : `POST /tokens/:tokenVerificationId/verify`
+    * verifyTokensWithMethod    : `POST /tokens/:tokenId/verifyWith`
```

Comment:
Yea this was intentional. The store procedure will look up the `tokenVerificationId` directly from the `tokenId`. The `tokenId` was needed because you need to set the verificationMethod on that session id as well.

Reply:
Cool 👍